pkgs/development/libraries/agda/cubical/default.nix

```diff
@@ -9,12 +9,14 @@ mkDerivation rec {
   src = fetchFromGitHub {
     repo = pname;
     owner = "agda";
-    rev = "v${version}";
-    sha256 = "07qlp2f189jvzbn3aqvpqk2zxpkmkxhhkjsn62iq436kxqj3z6c2";
+    rev = "d5030a9c89070255fc575add4e9f37b97e6a0c0c";
+    sha256 = "8WIrgAyA0bwFgtEjmwlvKQ7rZB91/Ppi8zOSq/qCTKE=";
   };
 
   LC_ALL = "en_US.UTF-8";
 
+  patches = [ ./no_werror_patch.txt ];
```

Comment:
This should have the same effect:

```
preConfigure = ''makeFlagsArray=("AGDA_EXEC=agda -W error")'';
```

Reply:
This would need to be the other way round I think

Reply:
I have already fixed it.